src/test/java/com/atlassian/bitbucket/jenkins/internal/link/BitbucketJobLinkActionFactoryTest.java

```diff
@@ -117,15 +127,6 @@ public void testCreateMultibranchSource() {
         assertThat(externalLink.getUrlName(), equalTo(BASE_URL + "/projects/PROJ/repos/repo/compare/commits?sourceBranch=refs%2Fheads%2Fbranch2"));
     }
 
-    @Test
```

Comment:
Did a manual test and verified that by using the head directly (which every job upon resolution has), we don't need to differentiate whether or not a pipeline step or checking out from the workflow job SCM. That relationship is no longer necessary, so this test is now covered by `testCreateMultibranchSource`.